headless-cms/rich-text-field-value-transformer/README.md

```diff
@@ -0,0 +1,35 @@
+# Transform Rich Text Field Value On Save
+
+This example shows how you can transform the form field's value before it's saved into the database. We took as an example the rich text field that uses Lexical Editor.
```

Comment:
> before it's saved into the database

I would say "before it's sent to the API", because this whole plugins is client-side.